tutorials/nnvm/from_darknet.py

```diff
@@ -105,24 +107,44 @@
 
 m.run()
 # get outputs
-out_shape = (net.outputs,)
-tvm_out = m.get_output(0).asnumpy().flatten()
+tvm_out = []
+if MODEL_NAME == 'yolov2':
+    layer_out = {}
+    layer_out['type'] = 'Region'
+    # Get the region layer attributes (n, out_c, out_h, out_w, classes, coords, background)
+    layer_attr = m.get_output(1).asnumpy()
+    layer_out['biases'] = m.get_output(2).asnumpy()
+    out_shape = (layer_attr[0], layer_attr[1]//layer_attr[0],
+                 layer_attr[2], layer_attr[3])
+    layer_out['output'] = m.get_output(3).asnumpy().reshape(out_shape)
+    layer_out['classes'] = layer_attr[4]
+    layer_out['coords'] = layer_attr[5]
+    layer_out['background'] = layer_attr[6]
+    tvm_out.append(layer_out)
+
+elif MODEL_NAME == 'yolov3':
+    for i in range(3):
+        layer_out = {}
+        layer_out['type'] = 'Yolo'
+        # Get the yolo layer attributes (n, out_c, out_h, out_w, classes, total)
+        layer_attr = m.get_output(i*4+1).asnumpy()
+        layer_out['biases'] = m.get_output(i*4+2).asnumpy()
+        layer_out['mask'] = m.get_output(i*4+3).asnumpy()
+        out_shape = (layer_attr[0], layer_attr[1]//layer_attr[0],
+                     layer_attr[2], layer_attr[3])
+        layer_out['output'] = m.get_output(i*4+4).asnumpy().reshape(out_shape)
```

Comment:
get_output[0] is always ignored?? if its ignored bcoz there as duplicates, then you can append outputs in order  [[output, mask, bias][output, mask, bias]] for different yolo layers
Otherwise out_memory will b double.

Reply:
Done